deploy/kessel-relations-deploy.yaml

```diff
@@ -9,7 +9,7 @@ objects:
       name: ${CLOWDAPP_NAME}-spicedb
     spec:
       config:
-        logLevel: debug
+        logLevel: ${SPICEDB_LOG_LEVEL}
```

Comment:
Are you sure it's not double braces here?  (It is for `SPICEDB_REPLICAS` and it's in the same parameters section.)

Reply:
Double braces i think are for integers and not strings.

Reply:
Ok, cool, if you are happy, please resolve. :)

Reply:
Otherwise LGTM.